codegen_glibmm/__init__.py

```diff
@@ -22,7 +22,7 @@
 import os
 
 # Update this as the version changes
-version_info = (0, 1, 0)
```

Comment:
Wouldn't it be enough just changing the major version?
Feels a bit odd to change both major and minor.

Reply:
or just minor maybe

Reply:
Because we are walking towards version 2, but we are not there yet. I wanted to call it 2.0.alpha1, but python setuptools only accepts numbers in the version_info, that's why I went for 1.99.0